object/token.go

```diff
@@ -272,75 +273,37 @@ func GetOAuthToken(grantType string, clientId string, clientSecret string, code
 		}
 	}
 
-	if grantType != "authorization_code" {
+	//Check if grantType is allowed in the current application
+	if !checkMethodValid(grantType, application.GrantTypes) {
 		return &TokenWrapper{
-			AccessToken: "error: grant_type should be \"authorization_code\"",
+			AccessToken: "error: grant_type does not support in this application",
```

Comment:
show grant type value, like ``fmt.Sprinf("error: grant_type: %s is not supported in this application", grantType)``